src/components/scenes/Fio/FioRequestListScene.tsx

```diff
@@ -395,25 +396,31 @@ class FioRequestList extends React.Component<Props, LocalState> {
     const currencyCode = pendingRequest.content.token_code.toUpperCase()
 
     const parsedUri = await currencyWallet.parseUri(pendingRequest.content.payee_public_address, currencyCode)
-    const guiMakeSpendInfo = {
+    const { pluginId } = currencyWallet.currencyInfo
+    const tokenId = getTokenId(account, pluginId, currencyCode)
+
+    const memos: EdgeMemo[] | undefined = parsedUri.uniqueIdentifier != null ? [{ type: 'text', value: parsedUri.uniqueIdentifier }] : undefined
+    const sendParams: SendScene2Params = {
+      walletId,
       fioPendingRequest: pendingRequest,
-      fioAddress: pendingRequest.payee_fio_address,
-      publicAddress: parsedUri.legacyAddress || parsedUri.publicAddress,
-      nativeAmount,
-      currencyCode,
-      metadata: parsedUri.metadata,
-      uniqueIdentifier: parsedUri.uniqueIdentifier,
-      spendTargets: [
-        {
-          nativeAmount,
-          publicAddress: parsedUri.legacyAddress || parsedUri.publicAddress,
-          otherParams: {
-            uniqueIdentifier: parsedUri.uniqueIdentifier,
-            fioAddress: pendingRequest.payee_fio_address
+      tokenId,
+      spendInfo: {
+        metadata: parsedUri.metadata,
+        spendTargets: [
+          {
+            nativeAmount,
+            publicAddress: parsedUri.legacyAddress || parsedUri.publicAddress,
```

Comment:
Do `parsedUri.legacyAddress ?? parsedUri.publicAddress`.

Reply:
For sure. was a copy pasta mistake